webviz_subsurface/plugins/_tornado_plotter_fmu/views/_tornado_widget.py

```diff
@@ -0,0 +1,320 @@
+import json
+from pathlib import Path
+from typing import Dict, List, Optional, Tuple, Union
+
+import pandas as pd
+from dash import (
+    ClientsideFunction,
+    Input,
+    Output,
+    State,
+    callback,
+    callback_context,
+    clientside_callback,
+)
+from dash.exceptions import PreventUpdate
+from webviz_config import WebvizSettings
+from webviz_config.webviz_assets import WEBVIZ_ASSETS
+from webviz_config.webviz_plugin_subclasses import ViewABC
+
+import webviz_subsurface
+from webviz_subsurface._components.tornado._tornado_bar_chart import TornadoBarChart
+from webviz_subsurface._components.tornado._tornado_data import TornadoData
+from webviz_subsurface._components.tornado._tornado_table import TornadoTable
+
+from .._plugin_ids import PlugInIDs
+from ..view_elements import Label, TornadoViewElement
+
+
+class TornadoWidget(ViewABC):
```

Comment:
Why not calling the class (and file) `TornadoView` instead of `TornadoWidget`?

Reply:
I just called it widget because that was the name in the old plugin, but I agree that view is better and clearer:)